### PR TITLE
hw7 after ENUM

### PR DESCRIPTION
**Pull request by Tabernol**

### CODE REVIEW

File: hw7/TankVolume.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    HUGE(380);
+
+    private double volume;
+    TankVolume(double volume) {
+        this.volume = volume;
+    }
+
+    public double getTankVolume(){
+        return volume;
+    }
```
Comment (shaitanaleksander) [inline hw7/TankVolume.java:16]: enum is not for data storage,  you don't need get and set method, just make  var  public final

File: hw7/Car.java [hunk 2]
```diff
@@ -104,27 +107,31 @@ public void showOdometer() {
...
     }
 
-    public double getLevelFuel() {
+    private double getLevelFuel() {
         return levelFuel;
     }
 
-    public void setLevelFuel(double levelFuel) {
+    private void setLevelFuel(double levelFuel) {
         this.levelFuel = levelFuel;
```
Comment (shaitanaleksander) [inline hw7/Car.java:135]: private get and set ? usually get and set are public

File: hw7/Car.java [hunk 3]
```diff
@@ -1,28 +1,31 @@
 package hw7;
 
-public class Car extends Transport  implements AirConsumptionAble {
-    public String name;
-    public double tankVolume;
-    public double engineVolume;
-    double fullWeight;
-    double unladenWeight;
+public class Car extends Transport implements AirConsumptionAble {
+    private String name;
```
Comment (shaitanaleksander) [inline hw7/Car.java:4]: implements AirConsumptionAble, bad naming of interface may be => airCconsumable

File: hw7/Car.java (hunk 2 above)
Comment (Tabernol) [reply hw7/Car.java:135]: I need the levelFuel data not to be available from other classes, but other methods in this class may change this value.

File: hw7/Car.java (hunk 3 above)
Comment (Tabernol) [reply hw7/Car.java:4]: You right.

File: hw7/TankVolume.java (hunk 1 above)
Comment (Tabernol) [reply hw7/TankVolume.java:16]: ok

File: hw7/Car.java (hunk 2 above)
Comment (shaitanaleksander) [reply hw7/Car.java:135]: than you can directly change the variable without get and set methods

**Pull request merged**